wagtail/admin/templates/wagtailadmin/widgets/chooser.html

```diff
@@ -7,10 +7,10 @@
     Any element with the 'action-choose' class will open the page chooser modal
     when clicked.
 {% endcomment %}
-<div id="{{ attrs.id }}-chooser" class="chooser {% block chooser_class %}page-chooser{% endblock %} {% if not value %}blank{% endif %}" {% block chooser_attributes %}{% endblock %}>
+<div id="{{ attrs.id }}-chooser" class="chooser {% block chooser_class %}{% if classname %}{{ classname }}{% endif %}{% endblock %} {% if not value %}blank{% endif %}" {% if chooser_url %}data-chooser-url="{{ chooser_url }}"{% endif %} {% block chooser_attributes %}{% endblock %}>
```

Comment:
Should the `data-chooser-url` be within the `chooser_attributes` block?

e.g.
```
{% block chooser_attributes %}{% if chooser_url %}data-chooser-url="{{ chooser_url }}"{% endif %} {% endblock %}
```

To closer align with the previous approach?

Reply:
I think it should, yes - good catch!

Reply:
Fixed and rebased now.